features/popover-hint.yml

```
@@ -0,0 +1,4 @@
+name: Popover=hint
```

For consistency with other features like this:

```suggestion
name: 'popover="hint"'
```